docs/content/en/docs/getting-started/site-engineer/_index.md

```diff
@@ -0,0 +1,232 @@
+---
+title: "K8ssandra site engineer quick start"
+linkTitle: "Site engineers"
+weight: 2
+description: |
+  Take a tour of your K8ssandra cluster using all the utilities!
+---
+
+**Completion time**: **10 minutes**.
```

Comment:
Add prerequisite that you've completed the first part of getting started, as was done in the developer quick start. Someone could get here by accident not having done the first part.

Reply:
Added the Important right at the top.